cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml

```diff
@@ -54,7 +54,13 @@ data:
         add_header Cache-Control "max-age=300";
         add_header Cache-Control "must-revalidate";
         add_header ETag "{{ $.Chart.Version }}";
-
+{{- if .Values.kubecostFrontend.tls }}
+{{- if .Values.kubecostFrontend.tls.enabled }}
+        ssl_certificate     /etc/ssl/certs/kc.crt;
+        ssl_certificate_key /etc/ssl/certs/kc.key;
+        listen 443 ssl;
```

Comment:
Consider moving `listen 9090;` to an else block here to prevent http access on 9090.